transformer_lens/loading_from_pretrained.py

```diff
@@ -2691,3 +2692,159 @@ def get_basic_config(model_name: str, **kwargs) -> Config:
+        "gpt2-small-attz-kk-L0"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L1_Hcat_z_lr1.20e-03_l18.00e-01_ds24576_bs4096_dc1.00e-06_rsanthropic_rie25000_nr4_v5": [
+        "gpt2-small-attz-kk-L1"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L2_Hcat_z_lr1.20e-03_l11.00e+00_ds24576_bs4096_dc1.00e-06_rsanthropic_rie25000_nr4_v4": [
+        "gpt2-small-attz-kk-L2"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L3_Hcat_z_lr1.20e-03_l19.00e-01_ds24576_bs4096_dc1.00e-06_rsanthropic_rie25000_nr4_v9": [
+        "gpt2-small-attz-kk-L3"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L4_Hcat_z_lr1.20e-03_l11.10e+00_ds24576_bs4096_dc1.00e-06_rsanthropic_rie25000_nr4_v7": [
+        "gpt2-small-attz-kk-L4"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L5_Hcat_z_lr1.20e-03_l11.00e+00_ds49152_bs4096_dc1.00e-06_rsanthropic_rie25000_nr4_v9": [
+        "gpt2-small-attz-kk-L5"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L6_Hcat_z_lr1.20e-03_l11.10e+00_ds24576_bs4096_dc1.00e-06_rsanthropic_rie25000_nr4_v9": [
+        "gpt2-small-attz-kk-L6"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L7_Hcat_z_lr1.20e-03_l11.10e+00_ds49152_bs4096_dc1.00e-06_rsanthropic_rie25000_nr4_v9": [
+        "gpt2-small-attz-kk-L7"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L8_Hcat_z_lr1.20e-03_l11.30e+00_ds24576_bs4096_dc1.00e-05_rsanthropic_rie25000_nr4_v6": [
+        "gpt2-small-attz-kk-L8"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L9_Hcat_z_lr1.20e-03_l11.20e+00_ds24576_bs4096_dc1.00e-06_rsanthropic_rie25000_nr4_v9": [
+        "gpt2-small-attz-kk-L9"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L10_Hcat_z_lr1.20e-03_l11.30e+00_ds24576_bs4096_dc1.00e-05_rsanthropic_rie25000_nr4_v9": [
+        "gpt2-small-attz-kk-L10"
+    ],
+    "ckkissane/attn-saes-gpt2-small-all-layers/gpt2-small_L11_Hcat_z_lr1.20e-03_l13.00e+00_ds24576_bs4096_dc3.16e-06_rsanthropic_rie25000_nr4_v9": [
+        "gpt2-small-attz-kk-L11"
+    ],
+    "nev/gpt2_xl_saes-saex-test/sae_weights": ["gpt2-xl-saex-resid-pre-l20"],
+}
+
+
+def make_sae_alias_map():
+    """
+    Equivalent of make_model_alias_map for SAEs.
+
+    Converts OFFICIAL_SAE_NAMES (the list of actual SAE names on
+    HuggingFace) and SAE_ALIASES (a dictionary mapping official SAE names to
+    aliases) into a dictionary mapping all aliases to the official SAE name.
+
+    Unlike make_model_alias_map does not do any .lower casting.
+    """
+    sae_alias_map = {}
+    for official_sae_name in OFFICIAL_SAE_NAMES:
+        aliases = SAE_ALIASES.get(official_sae_name, [])
+        for alias in aliases:
+            sae_alias_map[alias] = official_sae_name
+        sae_alias_map[official_sae_name] = official_sae_name
+    return sae_alias_map
+
+
+def convert_ckkissane_sae_cfg(ckkisane_sae_cfg: dict) -> HookedSAEConfig:
```

Comment:
typo: "ckkisane_sae_cfg" should be "ckkissane_sae_cfg". Same typo in get_pretrained_sae_config